readme.md

```diff
@@ -381,9 +381,11 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 
 - [normalize-url](https://github.com/sindresorhus/normalize-url) - Normalize a URL.
 - [humanize-url](https://github.com/sindresorhus/humanize-url) - Humanize a URL: http://sindresorhus.com → sindresorhus.com.
+- [url-unshort](https://github.com/nodeca/url-unshort) - Expand shortened URLs.
 - [speakingurl](https://github.com/pid/speakingurl) - Generate a slug from a string with transliteration.
 - [linkify-it](https://github.com/markdown-it/linkify-it) - Link patterns detector with full unicode support.
 - [url-pattern](https://github.com/snd/url-pattern) - Easier than regex string matching patterns for URLs and other strings.
+- [embedza](https://github.com/nodeca/embedza) - Create HTML snippets for whitelisted sites by URLs. Use oembed, opengraph and meta tags info.
```

Comment:
I still have no idea what you're thing is about just from the description. HTML snippets mean nothing to people.

Maybe:

> Create HTML snippets/embeds from URLs using info from oEmbed, Open Graph, meta tags.